app/models/invenio_rdm_record_converter.rb

```diff
@@ -525,10 +525,17 @@ def find_mapping_entry_in_dh_collections
           return mapping_entry
         # part of pediatric neurology brief collection in format pnb-volume#-issue#
         elsif collection_path_entry[:id]&.starts_with?("pnb-")
-          volume_number, issue_number = collection_path_entry[:id].split("-").last(2)
+          volume_number, issue_number = collection_path_entry[:id].split(/[^\d]/).reject(&:blank?)
+
+          if issue_number.present?
```

Comment:
This is the underlying problem - the issue number is not always present which causes errors on formatting